src/api/stats/fantom/getBeethovenxDualApys.js

```diff
@@ -65,9 +68,54 @@ const getFarmApys = async pools => {
     let yearlyRewardsInUsd = yearlyRewardsAInUsd.plus(yearlyRewardsBInUsd);
     const apy = yearlyRewardsInUsd.dividedBy(totalStakedInUsd);
     apys.push(apy);
+
+    let aprFixed = 0;
+    if (pool.staderls) {
+      aprFixed = await getLiquidStakingPoolYield(pool);
+    }
+    lsAprs.push(aprFixed);
+  }
+
+  return [apys, lsAprs];
+};
+
+const getLiquidStakingPoolYield = async pool => {
+  const balVault = getContractWithProvider(IBalancerVault, beethovenx.router, web3);
+  const tokenQtys = await balVault.methods.getPoolTokens(pool.vaultPoolId).call();
+
+  let qty = [];
+  let totalQty = new BigNumber(0);
+  for (let j = 0; j < tokenQtys.balances.length; j++) {
+    if (pool.composable) {
+      if (j != pool.bptIndex) {
+        const price = await fetchPrice({ oracle: 'tokens', id: pool.tokens[j].oracleId });
+        const amt = new BigNumber(tokenQtys.balances[j])
+          .times(price)
+          .dividedBy([pool.tokens[j].decimals]);
+        totalQty = totalQty.plus(amt);
+        qty.push(amt);
+      }
+    } else {
+      const price = await fetchPrice({ oracle: 'tokens', id: pool.tokens[j].oracleId });
+      const amt = new BigNumber(tokenQtys.balances[j])
+        .times(price)
+        .dividedBy([pool.tokens[j].decimals]);
+      totalQty = totalQty.plus(amt);
+      qty.push(amt);
+    }
+  }
+
+  let apr = 0;
+  const lsApr = 4.7;
+  try {
+    apr = (lsApr * qty[pool.lsIndex].dividedBy(totalQty).toNumber()) / 100;
+    apr = pool.balancerChargesFee ? apr / 2 : apr;
+  } catch (err) {
+    console.error(`Error fetching ls yield for ${pool.name}`);
```

Comment:
catch was needed in case network request fetching `lsApr` fails, not really needed for hardcoded one